README.md

```diff
@@ -223,3 +223,22 @@ This will run `eslint --fix` and automatically add changes to the commit. Please
   ]
 }
 ```
+
+### In-place minify the images before adding to the git repo
+
+```json
+{
+  "*.{png,jpeg,jpg,gif,svg}": [
+    "imagemin-lint-staged",
+    "git add"
+  ]
+}
+```
+
+<details>
+  <summary>More about `imagemin-lint-staged`</summary>
+
+  <a href="https://github.com/tomchentw/imagemin-lint-staged"><b>imagemin-lint-staged</b></a> is a CLI tool designed for lint-staged usage with sensible defaults.
```

Comment:
Any reason for using HTML instead of markdown here?

Reply:
From my VSCode preview, it seems markdown is not supported inside `<details>` as well. But if it's supported on GitHub, I'll revert it

Reply:
Yes, it is supported on Github.